Tribler/Core/TorrentChecker/torrent_checker.py

```diff
@@ -406,11 +422,8 @@ def _update_torrent_result(self, response):
         torrent_id = result[u'torrent_id']
         retries = result[u'tracker_check_retries']
 
-        # the result logic
-        is_good_result = seeders > 0 or leechers > 0
-
         # the status logic
-        if is_good_result:
+        if seeders > 0:
```

Comment:
I would be worried about crappy/intentionally crappy tracker responses. Not marking a torent as good will schedule it for removal. It has quite some consequences.